osu.Game/Overlays/BeatmapSet/Scores/TopScoreStatisticsSection.cs

```diff
@@ -111,10 +113,10 @@ public ScoreInfo Score
                 score = value;
 
                 accuracyColumn.Text = value.DisplayAccuracy;
-                maxComboColumn.Text = $@"{value.MaxCombo:N0}x";
+                maxComboColumn.Text = value.MaxCombo.ToLocalisableString(@"0\x");
 
                 ppColumn.Alpha = value.Beatmap?.Status.GrantsPerformancePoints() == true ? 1 : 0;
-                ppColumn.Text = $@"{value.PP:N0}";
+                ppColumn.Text = value.PP.ToLocalisableString(@"N0");
```

Comment:
Also is not `N0` web-side (compare e.g. https://osu.ppy.sh/beatmapsets/1193588#osu/2486881 - also no thousands spacing).